change/@fluentui-react-skeleton-50b8d438-835e-41ee-8cd3-503f3a0282d5.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Sorry to block, just noticed this is tagged as minor, this should be patch since you're not changing the API.
```suggestion
  "type": "patch",
```